core/pom.xml

```diff
@@ -19,12 +19,12 @@
 
   <parent>
     <groupId>io.zipkin.finagle2</groupId>
-    <artifactId>zipkin-finagle-parent_2.12</artifactId>
-    <version>22.4.0</version>
+    <artifactId>zipkin-finagle-parent_2.13</artifactId>
```

Comment:
hmm I thought in our build we did some tricks to dual publish to 12 and 13 via replacement?